src/components/static/Schedule.jsx

```diff
@@ -0,0 +1,92 @@
+"use client";
+import { useState, useEffect } from "react";
+
+const Schedule = () => {
+  const DAYS = [3, 4, 5, 6, 0];
+  const DAY_NAME = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+  ];
+
+  const [selectedDay, setSelectedDay] = useState(0);
+  // updates selected day based on current date (if applicable)
+  // this runs in an effect so there aren't unexpected hydration errors
+  useEffect(() => {
+    const today = new Date().getDay();
+    if (DAYS.includes(today)) {
+      setSelectedDay(today);
+    }
+  }, []);
+
+  const PLACEHOLDER_EVENTS = [...new Array(10)].map(() => ({
```

Comment:
should be pulling from bearhack calendar, make sure to make that api call server side with server component and feed it into this client component